### PR TITLE
fix(css): fix css grid in wallet > receive

### LINKED ISSUE #532: Wrappers/cards expand too much when few payment requests
**Describe the bug**
Wrappers/cards expand too much when there are few payment requests.

**To Reproduce**
1. Create a new wallet (not prefilled)
2. Go to the "Receive" tab

**Screenshots**
image

**Desktop (please complete the following information):**
 - Manjaro Hakoila

### PR DESCRIPTION
**Pull request by anler**
# PR Prelude

Thank you for contributing to sheikah! :)

**Please fill in this template (by putting an `x` inside
the brackets) _before_ filing your PR:**

- [X] I have read and understood [CODE_OF_CONDUCT][code] document.
- [X] I have read and understood [CONTRIBUTING][cont] document.
- [X] I have read and understood [STYLEGUIDE][style] document.
- [X] I have included tests for the changes in my PR. If not, I have included a
  rationale for why I haven't.
- [X] My code is formatted and is accepted by `yarn fmt`.
- [X] **I understand my PR may be closed if it becomes obvious I didn't
  actually perform all of these steps.**

# Why this change is necessary and useful

[Please explain **in detail** why the changes in this PR are needed.]

Fix a problem with the css grid in `Wallet / Receive` section.

With an empty wallet, if you navigate to Wallet > Send, the button for creating a new address is not aligned with the bottom of the card containing the form (because the card grows more than its content). Also, if a new address is created, the same happens with the card containing the address that is displayed below the card containing the form.

Fixes #532 

[code]: https://github.com/witnet/sheikah/blob/master/.github/CODE_OF_CONDUCT.md
[cont]: https://github.com/witnet/sheikah/blob/master/.github/CONTRIBUTING.md
[style]: https://github.com/witnet/sheikah/blob/master/docs/STYLEGUIDE.md

### CODE REVIEW

File: app/renderer/ui/components/wrapper/style.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
   .layout {
     display: grid;
     grid-template-columns: 1fr;
-    grid-template-rows: auto 1fr;
+    grid-template-rows: auto auto 1fr;
```
Comment (anler) [inline app/renderer/ui/components/wrapper/style.scss]: @Tommytrg warns us: **This is a hack!**

**Comment by aesedepece** [comment]
What's the effect of this? :see_no_evil:

**Comment by anler** [comment]
the bug manifests when you have an empty wallet, have you tried with that?

**Comment by aesedepece** [comment]
Merged as 5d6bbca3dca169eefe4bcbd2f4c9a55b6fdd0fee.

**Comment by aesedepece** [comment]
Awesome. A million thanks!!! :D

**Pull request closed without merging**